babel.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   presets: [['@vue/app', { useBuiltIns: 'entry' }]],
-  plugins: ['@babel/plugin-proposal-nullish-coalescing-operator']
+  plugins: [
+    '@babel/plugin-proposal-nullish-coalescing-operator',
+    '@babel/plugin-proposal-optional-chaining'
```

Comment:
This was added back in to fix an issue parsing an upgrade of one of the graphql packages (`graphql-language-service`)